src/components/organisms/DustStorm/DustStorm.tsx

```diff
@@ -42,6 +42,7 @@ const PotLuck: React.FC<PotLuckProps> = ({ openVenues, afterSelect }) => {
     afterSelect();
 
     if (IFRAME_TEMPLATES.includes(randomVenue?.template))
+      // @debt Refactor to use default spaceSlug, if needed.
       history.push(`/in/${randomVenue.id}`);
```

Comment:
[NOTE]
Most likely will break. Should be using
`VENUE_INSIDE_PARAM_URL` or at least `VENUE_INSIDE_BASE_URL` but probably through `venueInsideFullUrl()`

Reply:
It will definitely break, but this is burning man code. Should I even bother?

Reply:
Should you bother? 🤷
At least a `@debt` comment and/or ping @sparkletown/product